src/main/resources/templates/authorize.html

```diff
@@ -29,13 +29,13 @@ <h4 class="modal-title">Confirm Access</h4>
                                th:value="true" checked="checked"/>
                     </label>
                 </div>
-                <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}" />
+<!--                <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}" />-->
```

Comment:
Can this be removed?

Reply:
I believe it was necessary, but I was trying a lot of configurations at that time

This page will be moved to hydra relatively soon, so it may not be worth the effort